test/index.js

```diff
@@ -194,4 +195,39 @@ test('defaultComputeFileName', (t) => {
 	})
 })
 
+pTest('formatOptions are passed to sharp', co.wrap(function* (t) {
+	const file = yield vFile.read(src)
+
+	const file1 = file.clone()
+	const scale1 = {
+		format: 'webp',
+		maxWidth: 100,
+		formatOptions: {
+			quality: 80
+		},
+		maxWidth: Number.MAX_SAFE_INTEGER
+	}
+	const defaultFileSize = yield new Promise((res) => resize(file1, scale1, (err, newFile1) => {
+		t.ifError(err)
```

Comment:
Use `reject` from `new Promise((resolve, reject) => {...})` here. The whole test will abort then.

Reply:
Didn't get it. Should I just do:
```js
err && rej(err)
```
?

Reply:
Yes, put please like `if (err) return reject(err)` for readability.